parlai/core/torch_generator_agent.py

```diff
@@ -773,13 +804,22 @@ def __init__(self, beam_size, min_length=3, padding_token=0, bos_token=1,
         self.n_best_counter = 0
         self.min_n_best = min_n_best
         self.block_ngram = block_ngram
-        self.partial_hyps = [[self.bos] for i in range(beam_size)]
+        self.partial_hyps = [[] for i in range(beam_size)]
```

Comment:
Does bos still get inserted as an input? I think seq2seq and transformer/generator have different behaviors w.r.t. this.